controller/main_controller.php

```diff
@@ -64,38 +66,30 @@ public function display($route)
 		$this->user->add_lang_ext('phpbb/pages', 'pages_controller');
 
 		// Load the page data to display
-		$display = $this->load_page_data($route);
-
-		// Set the page title to use
-		$page_title = ($display) ? $display->get_title() : $this->user->lang('INFORMATION');
+		$page = $this->load_page_data($route);
 
 		// Assign the page data to template variables
 		$this->template->assign_vars(array(
-			'PAGE_TITLE'	=> $page_title,
-			'PAGE_CONTENT'	=> ($display) ? $display->get_content_for_display() : $this->user->lang('PAGE_NOT_AVAILABLE', $route),
+			'PAGE_TITLE'	=> $page->get_title(),
```

Comment:
Lets not use this twice, let's store this to a property (or variable with local scope) when we load page data then access that property/var?